install/load-deployment-reproduce.yaml

```
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pilot-load-kubeconfig
+  namespace: pilot-load
+data:
+  kubeconfig.yaml: |
+    apiVersion: v1
+    clusters:
+    - cluster:
+        server: http://apiserver.pilot-load:18090
+      name: load
+    contexts:
+    - context:
+        cluster: load
+        user: fake
+      name: load
+    current-context: load
+    kind: Config
+    preferences: {}
+    users:
+    - name: fake
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pilot-load-reproduce
+  namespace: pilot-load
+  labels:
+    app: pilot-load-reproduce
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: pilot-load-reproduce
+      annotations:
+        sidecar.istio.io/inject: "false"
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8765"
+      labels:
+        app: pilot-load-reproduce
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              namespaces: [istio-system]
+              labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - istiod
+              topologyKey: kubernetes.io/hostname
+      # May take a while to shut down
+      terminationGracePeriodSeconds: 2000
+      imagePullSecrets:
+      - name: gcr-io
+      containers:
+      - name: pilot-load
+        image: gcr.io/howardjohn-istio/pilot-load
+        imagePullPolicy: Always
+        command: [ "/bin/sh", "-ec", "while :; do echo '.'; sleep 5 ; done" ]
```

```suggestion
        command:
        - sleep
        - infinity
```

Seems to be the canonical way to do this in k8s

---

Done, Thank you.